<launch>
    <rosparam file="$(find spear_rover)/config/hardware.yaml" command="load"/>
    <rosparam file="$(find spear_rover)/config/controllers.yaml" command="load" />
    <rosparam file="$(find spear_rover)/config/joint_limits.yaml" command="load" />
    <node name="CASE_hardware_interface" pkg="spear_rover" type="CASE_hardware_interface_node" output="screen" />
    <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" ns="/"
        args="
            /CASE/controller/state
            /CASE/controller/velocity/left_front_wheel
            /CASE/controller/velocity/left_rear_wheel
            /CASE/controller/velocity/right_front_wheel
            /CASE/controller/velocity/right_rear_wheel
        " />

    <include file="$(find spear_rover)/launch/rover2_can.launch">
        <arg name="interface" value="vcan0"/>
    </include>

</launch>
